app/enterprise/1.5.x/kong-for-kubernetes/install.md

```diff
@@ -1,33 +1,32 @@
 ---
-title: Installing Kong for Kubernetes Enterprise
+title: Installing Kong Enterprise with Kubernetes
```

Comment:
Please split out the page into two separate pages, else this page is going to cause more problems. I myself got confused and had to scroll up and down a few times.